### PR TITLE
feat(php): convert PHP fully to helm deployments

### PR DESCRIPTION
**Pull request by hero-david**

### CODE REVIEW

File: .github/workflows/php.yml [hunk 1]
```diff
@@ -120,30 +90,6 @@ jobs:
...
-        if: github.event.inputs.CONTAINER != ''
-        env:
-          BUILD: ${{ github.run_number }}
-        run: |
-          echo "build_num=$(($BUILD+10000))" >> $GITHUB_OUTPUT
-          echo "image_tag_build=${{env.CONTAINER_REGISTRY}}/${{ inputs.CONTAINER }}-${{matrix.container}}:$(($BUILD+10000))" >> $GITHUB_OUTPUT
-          echo "image_tag_branch=${{env.CONTAINER_REGISTRY}}/${{ inputs.CONTAINER }}-${{matrix.container}}:${{github.ref_name}}" >> $GITHUB_OUTPUT
-      # If we do not receive a container name, use the repo name
-      - name: "Export build tag"
-        id: export_build_tag
```
Comment (matthewhartstonge) [inline .github/workflows/php.yml:138]: It looks like this specific step is required as part of the helm deploy step tag:

```
tag: ${{ steps.export_build_tag.outputs.build_num }}
```

**Review by matthewhartstonge — CHANGES_REQUESTED** [review]

**Review by matthewhartstonge — APPROVED** [review]
LGTM!

File: .github/workflows/php.yml (hunk 1 above)
Comment (matthewhartstonge) [reply .github/workflows/php.yml:138]: yey for copy+pasta :polite-cat:

Comment (hero-david) [reply .github/workflows/php.yml:138]: I mean, given that moving to helm only requires we have the build number (+1000...) and not the full image tag. We could set an output variable in build which takes either number output from either matrix, then passes that on....Might be worth a try in another branch?

Comment (matthewhartstonge) [reply .github/workflows/php.yml:138]: Yeah, might be worth a try if it cleans things up

Comment (hero-david) [reply .github/workflows/php.yml:138]: Would even be worth exporting to a re-usable action....Will add a ticket

**Comment by hero-david** [comment]
Print service cutover to helm with initial manual tag override and deployment. Merging this in so that we're all helm on PHP from now on :tada:

**Pull request merged**